graylog2-web-interface/src/components/welcome/Welcome.jsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import PageHeader from 'components/common/PageHeader';
+import SectionComponent from 'components/common/Section/SectionComponent';
+
+import LastOpenList from './LastOpenList';
+import PinnedItemsList from './PinnedItemsList';
+import RecentActivityList from './RecentActivityList';
+
+const FlexContainer = styled.div`
+  display: flex;
+  align-items: stretch;
+  flex-wrap: nowrap;
+  gap: 45px;
+`;
+
+const StyledSectionComponent = styled(SectionComponent)`
+  flex-grow: 1;
+`;
+
+const Welcome = () => (
+  <>
+    <PageHeader title="Welcome to Graylog">
+      <span>Here you can find most used content</span>
+    </PageHeader>
+    <FlexContainer>
```

Comment:
We can use the `SectionGrid` as a wrapper here, like we do it in similar cases. This way we use the case `gap` and responsive styles. Currently we do not change the grid for smaller screens, but it would look really good.